app/models/ability.rb

```diff
@@ -0,0 +1,18 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    can :read, Post
+
```

Comment:
- [ ] Please make sure to set the read value to all so the `users` can read all the entities(Post, Comment, Like) in the project. 
  - You can consider updating the code by this. `can :read, :all`